### PR TITLE
Update Lockfiles

### PR DESCRIPTION
**Pull request by karanataryn**
Ran `poetry lock` in various directories and fixed arising bugs: https://github.com/nltk/nltk/issues/3293. 135 packages were updated.

### CODE REVIEW

File: lib/sycamore/poetry.lock [hunk 1]
```diff
@@ -147,6 +162,48 @@ files = [
...
+[package.dependencies]
+numpy = ">=1.24.4,<2"
+opencv-python-headless = ">=4.9.0.80"
+tomli = ">=2.0.1"
+typing-extensions = ">=4.9.0"
+
+[[package]]
+name = "albumentations"
+version = "1.4.10"
+description = "An efficient library for image augmentation, providing extensive transformations to support machine learning and computer vision tasks."
```
Comment (HenryL27) [inline lib/sycamore/poetry.lock:185]: what's albumentations for?

File: lib/sycamore/poetry.lock [hunk 2]
```diff
@@ -5750,32 +5913,139 @@ virtualenv = ">=20.10.0"
...
+    {file = "prompt_toolkit-3.0.48-py3-none-any.whl", hash = "sha256:f49a827f90062e411f1ce1f854f2aedb3c23353244f8108b89283587397ac10e"},
+    {file = "prompt_toolkit-3.0.48.tar.gz", hash = "sha256:d6623ab0477a80df74e646bdbc93621143f5caf104206aa29294d53de1a03d90"},
 ]
 
 [package.dependencies]
 wcwidth = "*"
 
+[[package]]
+name = "propcache"
+version = "0.2.0"
```
Comment (HenryL27) [inline lib/sycamore/poetry.lock:5944]: what added this?

File: .github/workflows/testing.yml [hunk 3]
```diff
@@ -59,6 +59,9 @@ jobs:
       - name: Install sycamore
         run: poetry install --all-extras
         working-directory: lib/sycamore
+      - name: Download nltk packages
+        run: poetry run python -m nltk.downloader punkt_tab averaged_perceptron_tagger_eng
+        working-directory: lib/sycamore
```
Comment (HenryL27) [inline .github/workflows/testing.yml:64]: why do we need to do this twice?

**Review by HenryL27 — APPROVED** [review]
lgtm a few questions

File: .github/workflows/testing.yml (hunk 3 above)
Comment (karanataryn) [reply .github/workflows/testing.yml:64]: Once for unit test, once for integration. Was breaking otherwise.

File: lib/sycamore/poetry.lock (hunk 2 above)
Comment (karanataryn) [reply lib/sycamore/poetry.lock:5944]: The `yarl` package from the looks of it, which is required by `aiohttp`

File: lib/sycamore/poetry.lock (hunk 1 above)
Comment (karanataryn) [reply lib/sycamore/poetry.lock:185]: paddleocr needs it

**Pull request merged**